x-pack/legacy/plugins/canvas/public/legacy.ts

```diff
@@ -24,7 +22,16 @@ const shimCoreSetup = {
 const shimCoreStart = {
   ...npStart.core,
 };
-const shimSetupPlugins = {};
+const shimSetupPlugins = {
+  // When the frontend Canvas plugin is moved to NP, we can declare reporting as an optional dependency
+  // and enable exports based on existence of the plugin.
+  // For now, use an injected config to mimic this behavior in the meantime.
+  reporting:
+    npSetup.core && npSetup.core.injectedMetadata.getInjectedVar('reportingBrowserType')
+      ? {}
+      : undefined,
```

Comment:
The reason that Canvas had to check the Reporting browser type was because, if it was configured to `phantom`, the, the PDF would fail. Originally it was important to show that the configuration had to change and make the browser type become `chromium`

In Kibana 7.x we dropped support for phantom, and now chromium is the only valid option for the type setting! The setting will probably go away completely in 8.0.

Letting you know this, because you can remove lots of code in Canvas, since the browser type check isn't needed! 🍾